mantidproject/vesuvio
# Separate out calibration script into modules

**Description of work:**

This PR separates out the calibration script [calibrate_vesuvio_uranium_martyn_MK5.py](https://github.com/mantidproject/vesuvio/pull/61/files#diff-cd45feda55bf17d046f59e13c4c3e96339c643a4d6fd0759ff97f6cd4bdf4f97) into separate modules.

This script file was overly large and was getting hard to work with.

Whilst doing this, legacy formatting issues have been resolved (indentation is now 4 as per pep8 etc.).

Old versions of the script and system tests have been removed.

Given the separated scripts are now "harder" to use, a `README` has been added.

**To test:**

Using the new README as a guide:
1. Ensure system tests pass.
2. Ensure unit tests pass
3. Load the algorithms into `mantid` using the new script.



Resolves #62
